<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editBook</class>
 <widget class="QDialog" name="editBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit book</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>340</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>291</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLineEdit" name="edbName"/>
    </item>
    <item>
     <widget class="QLineEdit" name="edAut"/>
    </item>
    <item>
     <widget class="QSpinBox" name="edYear">
      <property name="maximum">
       <number>2222</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="genre">
      <item>
       <property name="text">
        <string>Ficion</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sci-fiction</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Thriller</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Romance</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Non-fiction</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fantasy</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>History</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Horror</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Children`s book</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edPublB"/>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="edRating">
      <property name="maximum">
       <double>10.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>Available</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Unavailable</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Book Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Author:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Year:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Genre:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Publisher:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Rating:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Status:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>editBook</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>editBook</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
